ui/scss/component/_claim-search.scss

```diff
@@ -26,9 +26,10 @@
 
 .claim-search__dropdown {
   padding: 0 var(--spacing-m);
-  padding-right: var(--spacing-l);
+  padding-right: 2.75rem;
```

Comment:
This 2.75 is pretty specific, none of the spacing vars worked? Is it because of the inherent size of the icon? Can that be calculated?

Reply:
I tried going the spacing vars route. Those could work, but the spacing isn't even. I think the ideal spacing lies somewhere between `spacing-l` and `spacing-xl`.

`spacing-l`:
<img width="656" alt="Screen Shot 2021-09-13 at 10 30 44 PM" src="https://user-images.githubusercontent.com/31634995/133468394-b8d9dafe-e8eb-4d4d-abeb-7dab2c5cfe12.png">

`spacing-xl`:
<img width="717" alt="Screen Shot 2021-09-13 at 10 31 36 PM" src="https://user-images.githubusercontent.com/31634995/133468376-de35b916-af63-4807-af88-e44e2c015277.png">

The width of the caret itself is roughly 0.875. Here's what the spacing would look like at 2.875 (1 + 0.875 + 1):

<img width="162" alt="Screen Shot 2021-09-15 at 11 36 51 AM" src="https://user-images.githubusercontent.com/31634995/133473818-5f2bb015-0d48-4749-a334-ba96ceb8d306.png">

I went with 2.75 because the text takes up a little extra space to the right (which makes the space to the left of the caret a bit bigger).

<img width="162" alt="Screen Shot 2021-09-15 at 11 51 05 AM" src="https://user-images.githubusercontent.com/31634995/133475914-6e3e492c-ee25-41bc-abee-345748762d03.png">

Interested to know what you think!

Reply:
I actually like -xl